src/main/asciidoc/_chapters/configuration.adoc

```diff
@@ -289,14 +289,14 @@ Use the following legend to interpret this table:
 link:https://hadoop.apache.org/cve_list.html[CVEs] so we drop the support in newer minor releases
 * icon:exclamation-circle[role="yellow"] = Not tested, may/may-not function
 
-[cols="1,2*^.^", options="header"]
+[cols="1,3*^.^", options="header"]
 |===
-| | HBase-1.7.x | HBase-2.3.x
-|Hadoop-2.10.x | icon:check-circle[role="green"] | icon:check-circle[role="green"]
-|Hadoop-3.1.0 | icon:times-circle[role="red"] | icon:times-circle[role="red"]
-|Hadoop-3.1.1+ | icon:times-circle[role="red"] | icon:check-circle[role="green"]
-|Hadoop-3.2.x | icon:times-circle[role="red"] | icon:check-circle[role="green"]
-|Hadoop-3.3.x | icon:times-circle[role="red"] | icon:check-circle[role="green"]
+| | HBase-1.7.x | HBase-2.3.x | HBase-2.4.x
```

Comment:
It looks like the contents of the HBase-2.3.x column and HBase-2.4.x column are identical. Instead of adding the new column, can we updated the existing "HBase-2.3.x" to be "HBase-2.3+" ?

Reply:
I agree that it would be better to keep to a minimum number of columns.

This might make it unclear that things are different for HBase 3.0+, but we can address that in a different PR.

Reply:
For me since 2.3.x is EOL while 2.4.x is still in maintanence, I prefer we make a separated column for 2.4.x. We can merge the columns for EOL release lines as they will not be changed in the future.

And another thing is that in the past we will remove the EOL release lines soon in this matrix, both for HBase and Hadoop, but for me I think we'd better keep it for some more time as lots of our users are still on the old releases, for example, my company still needs to support hadoop-2.6.0 for our customers...
Anyway, this could a another issue.

Reply:
oh I forgot that 2.3 is EOL. I'm expressly fine with this then. We should clean out the 2.3 stuff.

Reply:
As I said above, I plan to start a discuss thread about whether to keep EOL release for a little more time in our ref guide. SO we can file another issue for remove 2.3 related things in our ref guide as a whole, not only here.